dashboard/src/components/OperatorInstance/OperatorInstance.tsx

```diff
@@ -128,8 +128,21 @@ class OperatorInstance extends React.Component<IOperatorInstanceProps, IOperator
   }
 
   public render() {
-    const { isFetching, error, resource, csv, instanceName, namespace } = this.props;
+    const {
+      isFetching,
+      error,
+      resource,
+      csv,
+      instanceName,
+      csvName,
+      crdName,
+      namespace,
+      push,
+    } = this.props;
     const { resources } = this.state;
+    const onUpdateClick = () =>
+      push(`/operators-instances/ns/${namespace}/${csvName}/${crdName}/update/${instanceName}`);
```

Comment:
Starting to think it'd be worthwhile to define the URL template literals for UI routes in one place, and refer to them by name? (not necessarily now, but just to think about).

Reply:
you mean like the URLs that are already in `dashboard/src/shared/url.ts`? You are right, better not to add URL literals out of there.